hoot-core/src/main/cpp/hoot/core/io/BulkInsert.cpp

```diff
@@ -22,6 +22,6 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2016 DigitalGlobe (http://www.digitalglobe.com/)
```

Comment:
Given that the script is dependent on there being a git log, should we concern ourselves with the following scenario ... somehow the git log gets lost/corrupted/hit-by-a-train-in-Utah or some such disaster, would we care then if all copyright updates are then based off of the current year? (i.e. we would lose all prior years). The script could be tweaked to consider the years that are already in the copyright header and then reconcile them with the years pulled from the git log. This could be a whole lot of unnecessary concern though. Just wanted to throw that option out there like an Aaron Rodgers Hail Mary.